recipe/meta.yaml

```
@@ -1,58 +1,78 @@
 {% set name = "anaconda-cloud-auth" %}
-{% set version = "0.5.1" %}
+{% set version = "0.7.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/anaconda_cloud_auth-{{ version }}.tar.gz
-  sha256: 23b428b13e88c036905dc266a4b3c1dc35565e706341478659f316e45cd6c4dd
+  sha256: bdb49cdb5a22b5d0c1d825cf2b0529005628fa987b5a1466283d80722590044a
 
 build:
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
   number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
     - python
     - hatchling
     - hatch-vcs >=0.3
     - pip
+    - setuptools-scm >=7.1
   run:
     - python
+    - anaconda-cli-base >=0.4.0
+    - cryptography >=3.4.0
     - keyring
     - pkce
-    - python-dotenv
     - pydantic
     - pyjwt
+    - python-dotenv
     - requests
-    - cryptography >=3.4.0
     - semver <4
   run_constrained:
     - conda >=23.9.0
```

where did you get this one?
```suggestion
  run_constrained:
    - conda >=23.9.0
```

---

Charles added it for v0.5.1. I think it works only with `conda >=23.9.0`

---

okey